### PR TITLE
fix: Move message list scroll after OG image is loaded

### PR DESCRIPTION
**Pull request by HoonBaek**
### Description Of Changes

* Move the scroll of the message list after loading the OG message's thumbnail image

[UIKIT-3724](https://sendbird.atlassian.net/browse/UIKIT-3724)

[UIKIT-3724]: https://sendbird.atlassian.net/browse/UIKIT-3724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/smart-components/Channel/components/MessageList/index.tsx [hunk 1]
```diff
@@ -156,7 +156,7 @@ const MessageList: React.FC<MessageListProps> = ({
             return (
               <MessageProvider message={m} key={m?.messageId} isByMe={isByMe}>
                 <Message
-                  handleScroll={handleScroll}
```
Comment (sravan-s) [inline src/smart-components/Channel/components/MessageList/index.tsx:159]: I want to remove this prop ~ we need to probably move `handleMessageHeightChange` to ChannelProvider

Comment (sravan-s) [reply src/smart-components/Channel/components/MessageList/index.tsx:159]: or, better make messageList a UI component 
~brainstorming here

Comment (HoonBaek) [reply src/smart-components/Channel/components/MessageList/index.tsx:159]: Yeah
> move handleMessageHeightChange to ChannelProvider
This is a way that I was also considering

**Review by AhyoungRyu — APPROVED** [review]

**Review by sravan-s — APPROVED** [review]

**Comment by AhyoungRyu** [comment]
@HoonBaek I just merged my PR #511. Removing these conflicted files will resolve the conflict :)

**Pull request merged**